search/src/Components/dataset/dataset_summary.js

```diff
@@ -91,28 +86,21 @@ class DatasetSummary extends Component {
                 <IconButton aria-label="back to list" color="secondary" onClick={() => window.location.assign(baseURL + 'datasets')}>
                   <ListIcon />
                 </IconButton>
-                <Typography variant='h4' style={{fontWeight: 600}}>{getAttribute(this.state.metadata.info, "name")}</Typography>
+                <Typography variant='h4' style={{fontWeight: 600}}>{getAttribute(this.state.metadata, "name")}</Typography>
```

Comment:
note to self: we no longer need getAttribute here. This is reworked in #238 in any case.